Enumerables.rb

```diff
@@ -79,17 +80,18 @@ def my_map(&block)
     map
   end
 
-  def my_inject(result, &block)
+  def my_inject(&block)
+    result = self[0]
     if block_given? == true
-     my_each do |element|
-      result = block.call(result, element)
+     (1...self.length).my_each do |index|
+      result = block.call(result, self[index])
      end
     end
     result
   end
```

Comment:
Please there is still a bug in `my_inject` method.
Let me share with you what I get when testing it
```ruby
irb(main):014:0> [5,6,7,8,9,10].inject {|sum, n| sum + n }
=> 45
irb(main):015:0> [5,6,7,8,9,10].my_inject {|sum, n| sum + n }
=> 45
irb(main):016:0> [5,6,7,8,9,10].inject(5) {|sum, n| sum + n }
=> 50
irb(main):017:0> [5,6,7,8,9,10].my_inject(5) {|sum, n| sum + n }
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):17
        1: from Enumerables.rb:83:in `my_inject'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):002:0> [5,6,7,8,9,10].my_inject { |product, n| product * n }
=> 151200
irb(main):003:0> [5,6,7,8,9,10].inject { |product, n| product * n }
=> 151200
irb(main):004:0> [5,6,7,8,9,10].inject(1) { |product, n| product * n }
=> 151200
irb(main):005:0> [5,6,7,8,9,10].my_inject(1) { |product, n| product * n }
Traceback (most recent call last):
        3: from /home/elvisfosso/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        2: from (irb):5
        1: from Enumerables.rb:83:in `my_inject'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):006:0> 
```